core/src/main/java/org/infinispan/configuration/cache/CacheMode.java

```diff
@@ -18,41 +19,49 @@
    /**
     * Data is not replicated.
     */
+   @ProtoEnumValue(number = 1)
```

Comment:
We need to start from 0. According to proto3 spec (which we do not implement yet but we'll want to):  "For enums, the default value is the first defined enum value, which must be 0."  In proto2 this does not matter.

Reply:
Also, I see this needs to be fixed in other places: EventLogLevel, EventLogCategory, ExecutionMode, CounterState.   As I said, not a problem in proto2, but will save us a major headache when moving to 3.

Reply:
Yeah makes sense, lets get these things right before .Final